### DISCUSSION TITLE
Add "addChangeListener" to ktx-actors

### DISCUSSION THREAD

**Discussion by jgandert** [libktx/ktx#139] [Ideas]
Thanks for the libraries. @czyzby :+1: 

Here's a little addition I wrote to make it easier to write `ChangeListener` to button etc. events, since according to the Vis*Button these are recommended since they don't trigger the event when the button is disabled (contrary to ClickListener).

```kotlin
inline fun Actor.addChangeListener(crossinline changed: (event: ChangeListener.ChangeEvent, actor: Actor) -> Unit) {
    addListener(object : ChangeListener() {
        override fun changed(event: ChangeEvent, actor: Actor) {
            changed(event, actor)
        }
    })
}
```

Then one can add a listener like this:

```kotlin
addChangeListener { event, actor -> 
    println("ouch.. that hurt. Please don't click me again :(")
}
```

**Comment by czyzby** [comment]
It's already there in `ktx-actors` under `onChange`/`onChangeEvent`. See [events.kt](https://github.com/libktx/ktx/blob/master/actors/src/main/kotlin/ktx/actors/events.kt#L12-L38). Thanks for the suggestion though, don't hesitate to create a pull request next time. :+1:
